api/pyproject.toml

```diff
@@ -32,6 +32,8 @@ aiobotocore = "^1.3.3"
 rpy2 = "^3.4.5"
 redis = "^3.5.3"
 pyjnius = "^1.3.0"
+openpyxl = "^3.0.7"
```

Comment:
yes, pandas needs them in order to load Excel files

Reply:
I've added an item to our tech debt, for looking at getting rid of the xls file - it's just a really heavy way of doing things, but it's really fine for now, especially since we'd preferably be using an equation and not a lookup anyway!